templates/zerver/help/join-a-zulip-organization.md

```diff
@@ -0,0 +1,37 @@
+# Join a Zulip Organization
+
+On Zulip, an **organization** is one Zulip account associated with a
+ particular group or organization.
+
+1. To join a Zulip organization, first navigate to the URL of the Zulip server.
+
+2. Click the register button in the top right hand corner of the screen.
+
+    ![Register button](/static/images/help/register-button.png)
+
+3. You will be taken to the register page and prompted to **Enter your
+ work email address**. Enter your work email address and click **Sign Up**.
+
+
+4. You will then be taken to the page shown below, which asks you to confirm your
+ email address. Confirm your email address by clicking on
+ the confirmation link that will be emailed to you.
```

Comment:
Wait for the email to arrive in your inbox, and then click on the confirmation link in the email.